frontend/catalyst/pennylane_extensions.py

```diff
@@ -1734,6 +1735,47 @@ def circuit(x: float):
     return out_classical_tracer
 
 
+# Note that "print" would alias the built-in print function.
+def debug_print(x):
+    """A :func:`qjit` compatible print function for debugging.
+
+    Enables printing of numeric values at runtime. Can also print objects or strings as constants.
+
+    Args:
+        x: A single jax array whose numeric values are printed at runtime, or any object whose
```

Comment:
```suggestion
        x (jax.Arrar, str, Any): A single jax array whose numeric values are printed at runtime, or any object whose
```

Reply:
I would probably skip the explicit `str`, since that is handled the same as `Any`. What do you think?

Reply:
Sounds good to me!

Reply:
```suggestion
        x (jax.Arrar, Any): A single jax array whose numeric values are printed at runtime, or any object whose
```